CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+## [1.1.1](https://github.com/uphold/anonymizer/releases/tag/v1.1.1) (2019-10-29)
```

Comment:
These PRs haven't been assigned to a release yet. So this block should be removed for now.

Reply:
Is that what's tracked in https://github.com/uphold/github-changelog-generator/issues/13?

Reply:
Seems related. I'd replace an Unreleased section with simply not displaying the section at all, since our process doesn't include maintaining such a section.

Reply:
Sure, but my point is whether the changelog generator should generate such a section (with an incorrect heading, even) given that it has no explicit support for unreleased changes. Sounds like a bug, no? Eventually if we do support the Unreleased section, this content should appear there, but until then, the generator should omit it automatically, rather than force the user to manually remove it. Should I open a bug at https://github.com/uphold/github-changelog-generator?

Reply:
Yes, there's definitely room for improvement there. As it stands, the generator assumes it's being called when a new release is being done, which is not always the case.

Reply:
Ok, created https://github.com/uphold/github-changelog-generator/issues/72 to track that.